src/main/pages/che-7/extensions/che4z-installing.adoc

```diff
@@ -13,4 +13,46 @@ summary:
 
 :context: installing-che4z
 
-To download and install the Eclipse Che4z plugins, follow the instructions on link:https://projects.eclipse.org/projects/ecd.che.che4z/downloads[this page].
\ No newline at end of file
+To download and install the Eclipse Che4z plugins, follow the instructions on link:https://projects.eclipse.org/projects/ecd.che.che4z/downloads[this page]..Prerequisites
+- Minimum of 2 CPUs and 4GB RAM
+- Docker
+- Internet access
+
+
+.Procedure
+
+. Install Eclipse Che by running the following command. Replace `__<DATA_FOLDER>__` with the location of user data on the host machine:
++
+[subs="+quotes"]
+----
+$ docker run -it --rm -e CHE_MULTIUSER=true -e CHE_HOST=`hostname -f` -e \
+  CHE_WORKSPACE_VOLUME=/var/run/docker.sock:/var/run/docker.sock -v \
+  /var/run/docker.sock:/var/run/docker.sock -v /__<DATA_FOLDER>__:/data eclipse/che:7.0.0-beta-4.0 start
+----
++
+For more information on configuring Eclipse Che with Docker, see https://www.eclipse.org/che/docs/che-6/docker-multi-user.html[Installation on Docker].
```

Comment:
Che 6 docs have now been removed, so we shouldn't link to them.

Reply:
Is the content covering this point (configuring eclipse che with docker) now covered elsewhere or is it now deprecated?

Reply:
We do not support configuring Che with Docker anymore. You can install Che locally using minikube, minishift, microk8s or docker4mac/docker4windows kubernetes.